app/controllers/sessions_controller.rb

```diff
@@ -6,25 +6,25 @@ def new
 
   def create
     auth = request.env["omniauth.auth"]
-    user = User.where(:provider => auth['provider'],
-                      :provider_id => auth['uid'].to_s).first || User.create_from_omniauth!(auth)
+    user = User.where(provider: auth['provider'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.